Bug: Compiler error in 9.2.0 when using LV_LIBINPUT_XKB=1

### LVGL version

v9.2.0

### What happened?

Compilation of my project fails with errors:

```
Compiling .pio/build/native-tft-debug/libd71/lvgl/drivers/libinput/lv_xkb.c.o
In file included from .pio/libdeps/native-tft-debug/lvgl/src/drivers/libinput/lv_xkb_private.h:17,
                 from .pio/libdeps/native-tft-debug/lvgl/src/drivers/libinput/lv_xkb.c:10:
.pio/libdeps/native-tft-debug/lvgl/src/drivers/libinput/lv_xkb.h:36:18: error: unknown type name ‘lv_xkb_t’
   36 | bool lv_xkb_init(lv_xkb_t * dsc, struct xkb_rule_names names);
      |                  ^~~~~~~~
.pio/libdeps/native-tft-debug/lvgl/src/drivers/libinput/lv_xkb.h:42:20: error: unknown type name ‘lv_xkb_t’
   42 | void lv_xkb_deinit(lv_xkb_t * dsc);
      |                    ^~~~~~~~
.pio/libdeps/native-tft-debug/lvgl/src/drivers/libinput/lv_xkb.h:51:29: error: unknown type name ‘lv_xkb_t’
   51 | uint32_t lv_xkb_process_key(lv_xkb_t * dsc, uint32_t scancode, bool down);
      |                             ^~~~~~~~
*** [.pio/build/native-tft-debug/libd71/lvgl/drivers/libinput/lv_xkb.c.o] Error 1
```

The error can be fixed by adding
`#include "../../misc/lv_types.h"`
to file _src/drivers/libinput/lv_xkb.c_

![image](https://github.com/user-attachments/assets/ebbb3935-7dd6-4572-8a4f-eef0b50c7710)


### How to reproduce?

Build lvgl project with LV_USE_LIBINPUT=1 and LV_LIBINPUT_XKB=1

--- a/src/drivers/libinput/lv_libinput_private.h
+++ b/src/drivers/libinput/lv_libinput_private.h
@@ -18,7 +18,9 @@ extern "C" {
 
 #if LV_USE_LIBINPUT
 
+#if LV_LIBINPUT_XKB
 #include "lv_xkb_private.h"
+#endif
 
 /*********************
  *      DEFINES

--- a/src/drivers/libinput/lv_libinput_private.h
+++ b/src/drivers/libinput/lv_libinput_private.h
@@ -18,6 +18,8 @@ extern "C" {
 
 #if LV_USE_LIBINPUT
 
+#include "lv_xkb_private.h"
+
 /*********************
  *      DEFINES
  *********************/

--- a/src/drivers/libinput/lv_xkb.h
+++ b/src/drivers/libinput/lv_xkb.h
@@ -18,6 +18,7 @@ extern "C" {
 
 #if defined(LV_LIBINPUT_XKB) && LV_LIBINPUT_XKB
 
+#include "../../misc/lv_types.h"
 #include <stdbool.h>
 #include <xkbcommon/xkbcommon.h>
 

